src/content/core/endpoints/automatic-ssl.mdx

```diff
@@ -15,15 +15,7 @@ If you've configured Endpoints but they aren't appearing, please check our [FAQ
 
 ## Auto create ingresses from services
 
-Okteto can automatically create an SSL endpoint for your deployments. In order to take advantage of this feature, add the annotation below to your service's manifest:
-
-```yaml
-metadata:
-  annotations:
-    dev.okteto.com/auto-ingress: "true"
-```
```

Comment:
Are we sure we want to delete these examples? I guess their taking up a lot of room and not helping too much